src/mcp/tools/core/get_environment.ts

```diff
@@ -24,37 +25,53 @@ export const get_environment = tool(
   async (_, { projectId, host, accountEmail, rc, config }) => {
     const aliases = projectId ? getAliases({ rc }, projectId) : [];
     const geminiTosAccepted = !!configstore.get("gemini");
+    const projectFileExists = config.projectFileExists("firebase.json");
+    const detectedApps = await detectApps(process.cwd());
+    const allAccounts = getAllAccounts().map((account) => account.user.email);
+    const hasOtherAccounts = allAccounts.filter((email) => email !== accountEmail).length > 0;
+
+    const projectConfigPathString = projectFileExists
+      ? config.path("firebase.json")
+      : "<NO CONFIG PRESENT>";
+    const detectedAppsMap = detectedApps
+      .filter((app) => !!app.appId)
+      .reduce((map, app) => {
+        if (app.appId) {
+          map.set(app.appId, app.bundleId ? app.bundleId : "<UNKNOWN BUNDLE ID>");
+        }
+        return map;
+      }, new Map<string, string>());
+    const activeProjectString = projectId
+      ? `${projectId}${aliases.length ? ` (alias: ${aliases.join(",")})` : ""}`
+      : "<NONE>";
+    const acceptedGeminiTosString = geminiTosAccepted ? "Accepted" : "<NOT ACCEPTED>";
     return toContent(`# Environment Information
 
 Project Directory: ${host.cachedProjectDir}
-Project Config Path: ${config.projectFileExists("firebase.json") ? config.path("firebase.json") : "<NO CONFIG PRESENT>"}
-Active Project ID: ${
-      projectId ? `${projectId}${aliases.length ? ` (alias: ${aliases.join(",")})` : ""}` : "<NONE>"
-    }
+Project Config Path: ${projectConfigPathString}
+Active Project ID: ${activeProjectString}
+Gemini in Firebase Terms of Service: ${acceptedGeminiTosString}
 Authenticated User: ${accountEmail || "<NONE>"}
-Gemini in Firebase Terms of Service: ${geminiTosAccepted ? "Accepted" : "Not Accepted"}
-
-# Available Project Aliases (format: '[alias]: [projectId]')
-
-${dump(rc.projects).trim()}
-
-# Available Accounts:
-
-${dump(getAllAccounts().map((account) => account.user.email)).trim()}
+Detected App IDs: ${detectedAppsMap.size > 0 ? `\n\n${dump(Object.fromEntries(detectedAppsMap)).trim()}\n` : "<NONE>"}
+Available Project Aliases (format: '[alias]: [projectId]'): ${Object.entries(rc.projects).length > 0 ? `\n\n${dump(rc.projects).trim()}\n` : "<NONE>"}
+Available Accounts: ${hasOtherAccounts ? `\n\n${dump(allAccounts).trim()}` : "<NONE>"}
```

Comment:
Can we omit this header entirely if there are no other accounts logged in? Its a pretty rare use case in the first place.